<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>windowChat</class>
 <widget class="QWidget" name="windowChat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="textEditRead">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>530</width>
     <height>480</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>530</width>
     <height>32</height>
    </rect>
   </property>
   <widget class="QLabel" name="labelCut">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>0</y>
      <width>28</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelEmojy">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>2</y>
      <width>28</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelfile">
    <property name="geometry">
     <rect>
      <x>38</x>
      <y>2</y>
      <width>28</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMore">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>0</y>
      <width>28</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextEdit" name="textEditReadSend">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>590</y>
     <width>530</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>50</y>
     <width>170</width>
     <height>691</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtSend">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>700</y>
     <width>110</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtCloseLow">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>700</y>
     <width>90</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QWidget" name="widgetTop" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>721</width>
     <height>50</height>
    </rect>
   </property>
   <widget class="QLabel" name="labelNickName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>571</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCLose">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMax">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMin">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
